inc/REST/RemoteDataController.php

```diff
@@ -79,6 +79,6 @@ public static function execute_queries( WP_REST_Request $request ): array|WP_Err
 	}
 
 	public static function permission_callback(): bool {
-		return true;
+		return current_user_can( 'edit_post' );
```

Comment:
@pkevan Do you think we need to pass the current post ID here? This would require adding it to the payload?

Reply:
This doesn't improve the security stance, but might account for custom caps that only apply to certain post types or posts.

Reply:
By adding it to the payload here:
https://github.com/Automattic/remote-data-blocks/blob/23e571292a639043ea71e03bc142c10aa9b0420b/src/blocks/remote-data-container/hooks/useRemoteData.ts#L193